test/fake_modules/system_import_missing/index.js

```diff
@@ -0,0 +1 @@
+System.import('anyone');
```

Comment:
I suppose the `System.import` has been deprecated and we don't need to support it.